src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
Agree with @ljharb on this one for a few reasons:
1. Our usage of it requires different language support, which means that if we were to leave this as is, we're excluding anyone who uses a screen reader who doesn't speak English
2. The labels for the handles may not be useful as "slider handle". Not only is there no differentiation between the two handles, which is a little confusing, but there's also no indication of what that could mean when you move it. For instance, with the price slider, it'd be better to have the left slider be something like "minimum price slider handle" and the right slider be "maximum price slider handle". We could provide default text for the two sliders as something like "minimum slider handle" and "maximum slider handle", but we'd still need to have that be something that users can customize.

Additionally, you wouldn't really need to have the content title cased, though it likely doesn't make too much of a difference when read.